notify/notify.go

```diff
@@ -949,7 +949,7 @@ func (tms TimeMuteStage) Exec(ctx context.Context, l log.Logger, alerts ...*type
 		return ctx, alerts, nil
 	}
 
-	muted, err := tms.muter.Mutes(muteTimeIntervalNames, now)
+	muted, _, err := tms.muter.Mutes(muteTimeIntervalNames, now)
```

Comment:
We do not have `GroupMarker` (added [here](https://github.com/prometheus/alertmanager/pull/3792)) so there is nothing to be marked. Use the blank identifier here instead.